west.yml

```diff
@@ -114,7 +114,7 @@ manifest:
       groups:
         - hal
     - name: hal_silabs
-      revision: 1ec8dd99aa4ac3e8632d2aa28a7438049bb27102
```

Comment:
shouldn't hal changes come first to not break bisectability?

Reply:
Moved HAL changes to the beginning of the PR.